.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-07-09 23:34:13 +0200 using RuboCop version 0.41.2.
+# on 2016-10-13 17:03:27 +0530 using RuboCop version 0.43.0.
```

Comment:
@hferentschik , I ran the command rubocop --auto-gen-config which gave me new set of offenses. I think this will help to monitor the list of offenses we are doing. 
For example, with new auto-gen-config, we found new offenses like below Abc count increased to 17 from 13.